root/etc/e-smith/templates/etc/httpd/conf.d/nethserver.conf/10tls_policy_legacy

```diff
@@ -0,0 +1,8 @@
+{
+ my $tlsPolicy = $tls{'policy'} || 'legacy';
+ my $ciphersuite = ($httpd{SSLCipherSuite})? '' : '#';
+
+ if (($tlsPolicy eq 'legacy') || ($tlsPolicy eq '')) {
```

Comment:
Today I posted a [PR proposal](https://github.com/NethServer/nethserver-base/pull/102) to consistently use the empty string as "legacy" value. Here we could just check for ` ! $tlsPolicy`, that means: _apply legacy_.